spring-cloud-previews/README.md

```diff
@@ -4,7 +4,7 @@ Please see below for a list of additional preview starters provided, with depend
 
 These starters are not included in the Spring Framework on Google Cloud Bill of Materials (BOM), 
 and should be explicitly added as dependencies. For example, to use the starter for Cloud Natural Language, 
-add the following dependency to your `pom.xml`:
+add the following dependency to your `pom.xml`.
```

Comment:
Sorting entries in the table below pattern-matched on lines containing `:`, which caused issue and included this line of the readme into the sort. Replaced this punctuation as a quick fix, but will need to look into a replacement for the regex going forward.

Reply:
If we end up removing that conflicting `sed` line, we may just come back to use a colon here